src/index.ts

```diff
@@ -200,8 +220,20 @@ export class Amplitude {
    * Send an identify call containing user property operations to Amplitude servers.
    *
    * @param identifyInstance
+   * @param extra
    */
-  identify(identifyInstance: Identify): Promise<boolean> {
+  identify(
+    identifyInstance: Identify,
+    extra?: MiddlewareExtra,
+  ): Promise<boolean> {
+    const event = {
+      event_type: SpecialEventType.IDENTIFY,
+      user_properties: identifyInstance,
```

Comment:
In theory the user_properties are more for the payload inside identifyInstance?

Reply:
Maybe it is better to rename `user_properties`/`group_properties` to one of the:
* `user_identify`/`group_identify`
* `identify`/`group_identify`
* `identify`/`identify`

Reply:
@justin-fiedler I think, if a middleware wants to add/set/remove identify properties, `Identify` object is more preferred - it has API to add/set/remove them.

Reply:
I thought about that too but discussed with the team and we would prefer to keep the MiddlewarePayload pure data for now.

Reply:
fixed